index.html

```diff
@@ -1,16 +1,410 @@
+              </div>
+            </div>
+          </div>
+          <a
+            href="#carousel"
+            class="carousel-control-prev"
+            role="button"
+            data-slide="prev"
+          >
+            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+            <span class="sr-only">Previous</span> </a
+          ><a
+            href="#carousel"
+            class="carousel-control-next"
+            role="button"
+            data-slide="prev"
+          >
+            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+            <span class="sr-only">Next</span>
+          </a>
+        </div>
+      </section>
+
+      <section id="features">
+        <h2 class="display-4 text-center">
+          Features
+        </h2>
+        <div class="row">
+        <div class="col-12 col-sm-6 col-lg-3 d-flex flex-column align-items-center mb-5">
+        <i class="fas fa-cookie"></i></i>
+          <h3 class="h4 mb-4 text-center">Wedding Cakes</h3>
+        <p>No matter the ocassion, CAKE R US's got your back.No Shaking</p>
+        <a href="#" class="btn btn-sm btn-info mt-auto">Read More...</a></div>
+        
+        <div class="col-12 col-sm-6 col-lg-3 d-flex flex-column align-items-center mb-5"> <i class="fas fa-birthday-cake"></i><h3 class="h4 mb-4 text-center">Birthday Cakes</h3>
+        <p>
+          Birthday's are special occasion, let us know how we can CAKE it even
+          more special.
+        </p>
+        <a href="#" class="btn btn-sm btn-info mt-auto">Read More...</a></div>
+        
+        <div class="col-12 col-sm-6 col-lg-3 d-flex flex-column align-items-center mb-5">
+          <i class="fas fa-glass-cheers"></i>
+          <h3 class="h4 mb-4 text-center">Cakes for Occasions</h3>
+        <p>No matter the ocassion, CAKE R US's got your back.No Shaking</p>
+        <a href="#" class="btn btn-sm btn-info mt-auto">Read More...</a></div>
+        
+        <div class="col-12 col-sm-6 col-lg-3 d-flex flex-column align-items-center mb-5">
+          <i class="fas fa-glass-cheers"></i>
+          <h3 class="h4 mb-4 text-center">Custom Cakes</h3>
+        <p>
+          We have special offers for your wedding and can cater for any amount
+          of hungry guest.
+        </p>
+        <a href="#" class="btn btn-sm btn-info mt-auto">Read More...</a></div>
+        </div>
+      </section>
+      <section id="gallery">
+       <hr>
+       <main>
+         <h1 class="mb-sm-4 text-center">Cake Glorious Cake</h1>
+          <p class="lead text text-center mb-4">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec placerat mauris sed massa vehicula
+            </p>
+          
+          <div class="row">
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal1.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
```

Comment:
These alt tags could be more descriptive. What you want to do is describe the image, so something like 'butter cookie cake on plate' would be more appropriate.